setup.py

```diff
@@ -25,6 +27,13 @@
     "tabulate>=0.8.6,<0.9.0", 
 ]
 
+
+class OpsBuild(build_ext):
+    def run(self):
+        script = os.path.join(os.path.dirname(__file__), "finetune", "custom_ops", "build.sh")
+        if subprocess.run(["sh", script]).returncode != 0:
+            warnings.warn("Failed to build the finetune ops, most aspects of finetune should function anyway.")
```

Comment:
I would just be super explicit here: 

"Failed to build the finetune memory management ops required for use of `Scheduler`. If you don't intend to use `Scheduler` you can safely ignore this message.  To build the ops later, execute {script}"